app/views/home/_swap_form.html.haml

```diff
@@ -1,12 +1,14 @@
 .plain-pattern.border-bottom
   .container.text-center
     %p.choose-party
-      My preferred party is
+      What party you would really like to vote for ?
       = select :user, :preferred_party_id, nil, {}, {form: 'login-form'} do
         %option(disabled=true selected=true) ...
         = options_for_select(@parties.map {|p| [ p.name, p.id ]}, selected: @preferred_party_id)
     %p.choose-party
-      but I'm willing to vote for
+      When we find someone to vote for your party in
+      = election_constituency_other
+      what party could you vote for in exchange ?
```

Comment:
```suggestion
      which party could you vote for in exchange?
```

and needs a comma after the constituency.  But this question still feels like it needs simplifying somehow.

Reply:
Not saying it can't be simplified, but the reason it's relatively long is that it includes the explanation of what we're trying to do. And thus delivers the purpose and the question in one go. Rather than a separated explanation, followed by a question later on. (or worse, question followed by explanation, which is what we had).

Reply:
Yeah agreed, my point is that intuitively I feel there should be some shorter way of combining the purpose and the question.  I could be wrong of course.